cdc/sinkv2/tablesink/progress_tracker.go

```diff
@@ -118,3 +118,11 @@ func (r *progressTracker) minTs() model.ResolvedTs {
 
 	return r.lastMinResolvedTs
 }
+
+// trackingCount returns the number of pending events and resolved tss.
+// Notice: must hold the lock.
+func (r *progressTracker) trackingCount() int {
```

Comment:
We can add a `closed` flag into `progressTracker` to simplify `remove` and make `minTs()` more safe.